server/game/gameSystems/MoveCardSystem.ts

```diff
@@ -92,7 +92,6 @@ export class MoveCardSystem<TContext extends AbilityContext = AbilityContext> ex
               (!card.hasRestriction(EffectName.TakeControl, context) &&
                 !card.anotherUniqueInPlay(context.player))) &&
                 (!destination || context.player.isLegalLocationForCardType(card.type, destination)) &&
-                !EnumHelpers.isArena(card.location) &&
```

Comment:
This check prevents moving cards that are in play, I don't have any context about this so I'm not really sure if it's correct to remove this. The alternative would be to introduce a new system to move cards in play into the deck.